src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1109,14 +1109,24 @@
         "TELEMETRY": {
             "certs": {
                 "ca_crt": "/etc/sonic/telemetry/dsmsroot.cer",
-		"server_crt": "/etc/sonic/telemetry/streamingtelemetryserver.cer",
-		"server_key": "/etc/sonic/telemetry/streamingtelemetryserver.key"
-            },
-	    "gnmi": {
-		"client_auth": "true",
-		"log_level": "2",
-		"port": "50051"
-	    }
+                "server_crt": "/etc/sonic/telemetry/streamingtelemetryserver.cer",
+                "server_key": "/etc/sonic/telemetry/streamingtelemetryserver.key"
+            },
+            "gnmi": {
+                "client_auth": "true",
+                "log_level": "2",
+                "port": "50051"
+            }
+        },
+        "TUNNEL": {
+            "MuxTunnel0": {
```

Comment:
Same here, more entries to show\parse different possible configs.

Reply:
I'll add this and update as well